open-company/open-company-web
# Hateoas lockdown

card: https://trello.com/c/JD32qkHM

To test:
- check that the CI has no errors
- import all the companies
- check that on _ReadOnly_ you can't edit any section
- check that on _Buffer_ you can edit any section
